### ISSUE TITLE
[Bug] infinite loading screen bug

### ISSUE THREAD

**Issue by SeaSack** [LukeYui/EldenRingSeamlessCoopRelease#466]
### Discord Username

SeaSackPotatoes#7210

### Describe the issue in detail

 I have an infinite loading screen bug, i joined friends and they had the round table hold but I didnt have it unlocked so I went to my own session to unlock it but I got the infinite loading screen I alt f4 but its just the infinite loading screen.

### Steps to recreate the bug

three friends, I joined their session and they were at the round table hold, currently I shouldn't have been able to go there but I did. so after we fast traveled to castle morne to kill the boss there afterwards we went to fast travel back to round hold table, but it didn't show up on the map for me. so I went to my own session and rested at site of grace and selected the option to head to the round table hold like if it were my first time going there but, from there I got an infinite loading screen. I alt f4 but I still have an infinite loading screen.

### Version

1.4.3 (Latest)

**Comment by xue12301** [comment]
can confirm the bugs still happening even after update :C

**Comment by Bodaciouslycrazy** [comment]
Exact bug just happened to me. Joined a friend who had already unlocked roundtable. While playing in his world, Melina showed up at a site of grace to take me to the round table hold, but I declined as I wanted to keep playing with my friend.
The grace for the round table hold didn't show up in my map, and is unavalable, and the option for "Take me to the round table" in the site of grace menu wasn't showing up.

I decided to quit playing with him and see if I can unlock it myself in my own world. When resting at a site of grace, the option to take me to the round table finally appeared.
It started a loading screen, then when the loading screen finished it would just flash a blank black screen, then start another loading screen. Repeats infinitely.

Quitting and re-loading the game doesn't solve the issue, it just starts loading repeatedly again.
Completely soft locks your save file.

**Comment by TheJinxed** [comment]
Happened to me just today as well. Was playing with 2 friends, they had the Round Table, and I was on a fresh character. Host left while we were all afk for food to go sleep. It kicked me to my own saved world, and I proceeded to play till I caught up to their spot in the story. I got to where the others were saying, in Castle Morne when you first get the option to go there, and it loads in, there's a split second where the game fully loads, then it reloads once again. I'm using the latest version (1.5.1) of the mod, no other mods being used
